ui/src/plugins/en.json

```diff
@@ -65,6 +65,16 @@
   "registerYouth_2b": "YMCA LA",
```

Comment:
[YMCA of Metropolitan Los Angeles](https://www.ymcala.org/)****

should this be spelled out ( to avoid confusion with Louisiana :) )